---
title: "Spencer Lab - David's Bio"
layout: textlay
excerpt: "Spencer Lab -- David's Bio"
sitemap: false
permalink: /dhsbio/
---

## David H. Spencer MD, PhD

![Dave](/images/peoplepic/davebio.JPG){:width="250px" style="float:left;
margin-right:20px;"} David is from Iowa City, Iowa. He attended Iowa City City High
School where he played soccer and was introduced to human genetics
through a program that placed high school students in research labs at
the University of Iowa. He worked for geneticist Brian Schutte
PhD, and later spent two summers in the lab of Jeff Murray MD, where
he first learned how to sequence DNA and heard about the human genome project. After high school, David left Iowa for Seattle, Washington
to attend college at the University of Washington. He earned a  B.S. in Biochemistry and worked for four years in the UW Genome
Center studying the genome of the opportunistic gram-negative
bacterial pathogen, *Pseudomonas aeruginosa* , where he was a
co-author on the paper describing the *Pseudomonas* [reference genome](https://pubmed.ncbi.nlm.nih.gov/10984043/),
and sequencing and comparative analysis of isolates collected from [evironmental
sources and patients with cystic fibrosis](https://pubmed.ncbi.nlm.nih.gov/12562802/). After graduating, David
spent one year as a Research Technician in the lab of genome mapping pioneer Maynard V. Olson PhD before joining the UW MSTP training
program. His graduate studies took place in the new [Department of Genome
  Sciences](https://www.gs.washington.edu/) that was formed from the Departments of Genetics and
  Molecular Biotechnology with co-mentors Maynard and fellow human
  genome pioneer Bob Waterston (both of whom came to the UW from the
  Department of Genetics at Wash
  U). David's thesis work was on the development
  and application of methods to sequence and analyze large regions of
  the human genome. This included developing an efficient method for
  [identifying potentially pathogenic mutations](https://pubmed.ncbi.nlm.nih.gov/17033972/)
  using principles of population genetics (Kingman's coalescent), and
  one of the first studies to use [hybridization-capture to identify disease-causing mutations](https://pubmed.ncbi.nlm.nih.gov/19409521/) in the
  human genome. After completing his MD and PhD, David and his family
  moved to St. Louis for residency training in Clinical Pathology and
  a fellowship in Molecular Genetic Pathology at Washington
  University/Barnes-Jewish Hospital in St. Louis, Missouri. During
  this training, David helped to develop one of the first [clinical NGS
  assays](https://pubmed.ncbi.nlm.nih.gov/24211365/) for detecting cancer mutations. Following residency and
  fellowship, he did post-doctoral research in the laboratory of
  renowned cancer biologist Tim Ley, where he studied the genomics and
  epigenomics of acute myeloid leukemia (AML). He led studies that
  defined the effects of [decitabine treatment on DNA methylation](https://pubmed.ncbi.nlm.nih.gov/23297133/) in
  primary AML samples, identified [functional differences between AML
  subclones](https://pubmed.ncbi.nlm.nih.gov/24613412/), investigated the mechanism involved in [HOX gene
  regulation in normal hematopoiesis and AML](https://pubmed.ncbi.nlm.nih.gov/25600023/), and characterized the
  landscape of [DNA methylation changes](https://pubmed.ncbi.nlm.nih.gov/28215704/) that occur in primary AML
  samples with mutations in the de novo DNA methyltransferase DNMT3A.

David started his own lab at Wash U in 2016 that is focused on improving the
genetic diagnosis of AML and our understanding of its molecular
pathogenesis at the level of the genome. His lab studies the
regulation of the HOX gene family and how the chromatin regulator CTCF
organizes the three-dimensional structure of the HOX locus and
influences functional interactions with regulatory enhancers in normal
hematopoietic cells and in AML. He also studies patterns of DNA
methylation in AML and how these changes relate to recurrent
mutations, and the mechanisms by which altered DNA methylation
influences genome architecture and gene regulation. David directs the
CLIA-Licensed Environment clinical sequencing laboratory at the
[McDonnell Genome Institute](https://www.genome.wustl.edu/),
which supports clinical trials involving sequencing of tumor samples,
including AML. The laboratory also performs sequencing of patient
samples for clinical tests available at Washington University,
including clinical gene panel assays for solid tumors and myeloid
malignancies, whole exome sequencing of tumors for tumor vaccine
studies, and targeted sequencing for inherited diseases. David is also
a co-developer of [ChromoSeq](https://www.nejm.org/doi/full/10.1056/NEJMoa2024534), a clinical whole genome sequencing assay
for genomic profiling of AML and MDS, which is currently in use for
routine diagnostic testing of all AML patients at Barnes-Jewish
Hospital in St. Louis. His work has been supported by grants from the National Cancer Institute, the American Society of Hematology, the American Cancer Society, The Cancer Research Foundation, and The Doris Duke Charitable Foundation.
